ansible_navigator/utils.py

```diff
@@ -403,3 +347,70 @@ def str2bool(value: Any) -> bool:
         if value.lower() in ("no", "false"):
             return False
     raise ValueError
+
+
+def templar(string: str, template_vars: Mapping) -> Any:
+    """template some string with jinja2
+    always to and from json so we return an object if it is
+
+    :param string: The template string
+    :type: string: str
+    :param template_vars: The vars used to render the template
+    :type template_vars: dict
+    """
+    # hide the jinja that may be in the template_vars
+    template_vars = escape_moustaches(template_vars)
+
+    if HAS_ANSIBLE:
+        logger.info("Ansible installed, Ansible's plugins will be available")
+
+        ansible_templar = Templar(loader=None)
```

Comment:
Should we support Ansible plugins? This requires direct import from `ansible/ansible`

Reply:
No, this can be switched to just jinja for templating, I think jordan yaml PR should give us good looking doc examples......  this was really just added for the to_yaml filter plugin